python/pyspark/pandas/frame.py

```diff
@@ -5266,12 +5266,12 @@ def to_orc(
         **options: "OptionalPrimitiveType",
     ) -> None:
         """
-        Write the DataFrame out as a ORC file or directory.
+        Write a DataFrame to the ORC format.
 
         Parameters
         ----------
         path : str, required
-            Path to write to.
+            Path to write to. It's required in pandas API on Spark whereas optional in pandas.
```

Comment:
It says `path : str, required`. should we mention that it's required here? looks duplicated.